src/kakarot/accounts/library.cairo

```diff
@@ -25,6 +25,14 @@ from kakarot.interfaces.interfaces import IAccount
 func nonce() -> (nonce: felt) {
 }
 
+@storage_var
```

Comment:
I understand this is to work around events, should we add TODOs to delete them after event fix?

Reply:
we should! I was pretty sure to have some added, but apparently not